### PR TITLE
Fix image upload

### PR DESCRIPTION
**Pull request by lemieux**
Image upload wasn't working for multiple reasons : 1. bad data sent, 2. all message data was appended to the form when it wasn't needed.

This should fix it once the backed is also fixed.

### CODE REVIEW

File: src/frame/js/actions/conversation.js [hunk 1]
```diff
@@ -112,19 +113,19 @@ function postSendMessage(message) {
 
 function postUploadImage(message) {
     return (dispatch, getState) => {
-        const {config: {appId}, user: {_id}} = getState();
+        const {config: {appId}, user: {_id}} = getState();        
         const blob = getBlobFromDataUrl(message.mediaUrl);
-
         const data = new FormData();
+
         data.append('source', blob);
```
Comment (mspensieri) [inline src/frame/js/actions/conversation.js]: Can we put `source` as the last object to be appended? It gives some flexibility to optimize validation server-side later on

File: src/frame/js/index.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import './utils/polyfills';
+// import './utils/polyfills';
```
Comment (mspensieri) [inline src/frame/js/index.js]: ?

File: src/frame/js/actions/conversation.js [hunk 3]
```diff
@@ -112,19 +113,19 @@ function postSendMessage(message) {
 
 function postUploadImage(message) {
     return (dispatch, getState) => {
-        const {config: {appId}, user: {_id}} = getState();
+        const {config: {appId}, user: {_id}} = getState();
```
Comment (mspensieri) [inline src/frame/js/actions/conversation.js]: format gone wrong

File: src/frame/js/index.js (hunk 2 above)
Comment (lemieux) [inline src/frame/js/index.js]: ugh, leftover of a test

File: src/frame/js/actions/conversation.js (hunk 1 above)
Comment (lemieux) [inline src/frame/js/actions/conversation.js]: sure thing

File: src/frame/js/actions/conversation.js (hunk 3 above)
Comment (lemieux) [inline src/frame/js/actions/conversation.js]: waa

**Pull request merged**